language/dsl-lite/README.md

```diff
@@ -0,0 +1,156 @@
+# DSL-Lite
+
+🚧🚧🚧🚧 WIP 🚧🚧🚧🚧
+
+A lightweight JSX runtime implementation for writing [Player UI](https://player-ui.github.io/) experiences using familiar React-like syntax.
+
+This library enables developers to create Player UI content using JSX instead of writing raw JSON, leading to cleaner, more maintainable, and type-safe code.
+
+It's an option to `@player-tools/dsl` for scenarios where the user requires compilation speed and doesn't require other React API than the hooks exposed by this library.
+
+## Features
+
+- ✨ **React-like Syntax**: Write Player UI experiences using familiar JSX syntax
+- 🔄 **Hooks API**: Includes React-inspired hooks such as `useRef` and `useContext`
+- 🧩 **Component Model**: Create reusable UI components with props and composition
+- 🌲 **AST Generation**: Generate and manipulate structured abstract syntax trees
+- 📝 **Template Literals**: Convenience utilities for binding expressions and data
+- 🔍 **Type Safety**: Full TypeScript support with comprehensive type definitions
+
+## Installation
+
+```bash
+# Using npm
+npm install player-ui-jsx
+
+# Using yarn
+yarn add player-ui-jsx
+
+# Using pnpm
+pnpm add player-ui-jsx
+```
+
+## Usage
+
+### Basic Example
+
+```tsx
+import { render } from "player-ui-jsx";
+import { Text, Asset } from "player-ui-jsx/components";
```

Comment:
I don't think these are the right imports? Also do we want to do the subpath exports for the components or just top level all of them like we did before